app/build.gradle.kts

```diff
@@ -1,5 +1,3 @@
-@file:Suppress("DSL_SCOPE_VIOLATION")
```

Comment:
BTW after the gradle update it can be also removed from `build.gradle.kts`

Reply:
Thank you for pointing out. I've removed it also from mentioned `build.gradle.kts`